### ISSUE TITLE
if theme_mode : dark in chirpy 7.2.0 -- ToC, tooltip, mobile hamburger menu and post header image bugs

### ISSUE THREAD

**Issue by SoumyaK4** [cotes2020/jekyll-theme-chirpy#2066]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Describe the bug

At first I thought it's because of the code modifications so I created a fresh 7.2.0 to test this, so this error isn't related to upgrading from previous version.

1. Post header image not showing on home page post list and inside post as well.
2. Tooltip is showing but very weird.
3. ToC is not showing(both desktop and mobile)
4. The mobile hamburger is not clickable, I tested with 2 different phones
5.  Images inside posts looks like they're buffering. 
6. I've copied the Text and Typography post from the chirpy docs to showcase the problems: [link](https://chirpy.soumyak4.in/posts/demo-blog/)

Here is the screen record:
[Screencast from 2024-11-28 20-27-19.webm](https://github.com/user-attachments/assets/15cef2e9-fd4e-4daa-9a2e-70264ecc20af)

While recording

### Steps To Reproduce

1. in `_config.yml` if `theme_mode: dark`
2. Both locally and after [Hosting here](https://chirpy.soumyak4.in/)

### Expected Behavior

[Screencast from 2024-11-28 20-31-19.webm](https://github.com/user-attachments/assets/dfdbb2e3-047a-4b35-869b-401c3b500ebb)

### Environment

- Ruby: 
- Jekyll: 
- Chirpy:

**Comment by DmitriyFrogo** [comment]
Same
Wanted to create issue, but you were faster
Hope this screenshot helps

![ff-error](https://github.com/user-attachments/assets/5ca2ab6f-c634-449d-9495-e7fa41055e4a)

**Comment by lucasbral** [comment]
Same Issue Here.

**Comment by SoumyaK4** [comment]
I am not able to label this as a bug, so if someone knows please do it

**Comment by kungfux** [comment]
It looks like this simple change in `_includes/head.html` solves the problem, but it is not logical, so I will postpone with a pull request proposal.

```diff
-  {% unless site.theme_mode %}
    <script src="{{ '/assets/js/dist/theme.min.js' | relative_url }}"></script>
-  {% endunless %}
```

**Comment by cotes2020** [comment]
> ```diff
> -  {% unless site.theme_mode %}
>     <script src="{{ '/assets/js/dist/theme.min.js' | relative_url }}"></script>
> -  {% endunless %}
> ```

That's right, after refactoring `theme.js` in _v7.2.0_, this condition is no longer needed. Removing `unless` will fix this issue.
